src/anndata_to_bq.py

```diff
@@ -56,33 +57,36 @@ def process(input_file, cas_cell_index_start, cas_gene_index_start):
     adata.var['cas_gene_index'] = np.arange(cas_gene_index_start, cas_gene_index_start + len(adata.var))
     col_index_to_cas_gene_index = [None] * len(adata.var)
     for col in range(0, len(adata.var)):
-        col_index_to_cas_gene_index[col] = adata.var['cas_gene_index'].iloc[[row]][0]
+        col_index_to_cas_gene_index[col] = adata.var['cas_gene_index'].iloc[[col]][0]
```

Comment:
the actual bug: `row` was the iteration variable from the previous loop starting on line 47